<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-09T17:43:59 IST" name="Suite" finished-at="2024-01-09T17:44:05 IST" duration-ms="5999">
    <groups>
      <group name="Error Handling">
        <method signature="ErrorValidation.loginError()[pri:0, instance:FrameWork.tests.ErrorValidation@66d3eec0]" name="loginError" class="FrameWork.tests.ErrorValidation"/>
      </group> <!-- Error Handling -->
    </groups>
    <test started-at="2024-01-09T17:43:59 IST" name="Test" finished-at="2024-01-09T17:44:05 IST" duration-ms="5999">
      <class name="FrameWork.tests.ErrorValidation">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:FrameWork.tests.ErrorValidation@66d3eec0]" started-at="2024-01-09T17:43:59 IST" name="launchApplication" finished-at="2024-01-09T17:44:04 IST" duration-ms="5203" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginError()[pri:0, instance:FrameWork.tests.ErrorValidation@66d3eec0]" started-at="2024-01-09T17:44:04 IST" name="loginError" finished-at="2024-01-09T17:44:04 IST" duration-ms="403" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginError -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:FrameWork.tests.ErrorValidation@66d3eec0]" started-at="2024-01-09T17:44:04 IST" name="tearDown" finished-at="2024-01-09T17:44:05 IST" duration-ms="345" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- FrameWork.tests.ErrorValidation -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
